src/modules/umount/UmountJob.cpp

```diff
@@ -36,7 +36,7 @@ UmountJob::~UmountJob() {}
 QString
 UmountJob::prettyName() const
 {
-    return tr( "Unmount file systems." );
+    return tr( "Unmounting file systems…", "@status" );
```

Comment:
Is changing this to "Unmounting" always correct in all ways that `prettyName()` is used?

Reply:
Since this is a `Job` subclass, the aPI documentation in `Job.h` applies:

- `prettyName` is a human-readable name/description of what the job does (not what it is **doing**),
- `prettyDescription` is a longer humand-readable description used in the summary page
- `prettyStatusMessage` is a human-readable description of what the job is **doing** and can contain numbers / update information

There is some overlap between them, and some fallback mechanisms so you don't necessarily have to write all three (in particular, the description is rarely used because it's for things-that-will-be-show-in-the-summary-page).

This will look a bit weird in the logs, because the name is used to list which jobs will run. This will look less weird during installation, because this particular job does not have a description or status, so its name gets used instead. This PR would be better if it added `prettyStatusMessage()` and used this status-style message for that, leaving the original name unchanged.